packages/block-editor/src/hooks/layout.js

```diff
@@ -65,7 +82,7 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 	return (
 		<InspectorControls>
 			<PanelBody title={ __( 'Layout' ) }>
-				{ !! defaultLayout && (
+				{ layoutType.canInherit && !! defaultThemeLayout && (
```

Comment:
I don't think "inheritance depend on the layout type itself, of course using "flex" for the default layout is not great but in the future a "grid layout" could definitely be used. So I'm wondering whether we should just assume that all layouts can be inherited.

Reply:
I think blocks should be able to opt-out of inheritance though in the block support config maybe.